### PR TITLE
updated meta data and icons

### PR DESCRIPTION
**Pull request by DiabolusGX**
- Changed default icons
- Added favicon
- Updated meta tags
- Modified title

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
+    "short_name": "Backdoor",
+    "name": "Backdoor to Frigidsec",
```
Comment (AviusX) [inline public/manifest.json]: Change this to just "Backdoor" or something like "Backdoor - Cybersecurity forum"

File: public/manifest.json [hunk 2]
```diff
@@ -1,25 +1,19 @@
...
+        "type": "image/png"
     }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+    ],
+    "theme_color": "#E95678",
+    "background_color": "#ffffff",
```
Comment (AviusX) [inline public/manifest.json]: Change background_color to #16161C

File: public/manifest.json [hunk 3]
```diff
@@ -1,25 +1,19 @@
...
+        "sizes": "512x512",
+        "type": "image/png"
     }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+    ],
+    "theme_color": "#E95678",
```
Comment (AviusX) [inline public/manifest.json]: Change theme color to #F43E5C

Comment (DiabolusGX) [reply public/manifest.json:16]: I tried that color but I think it was too bright and the link embed color will not look good, so I picked lighter one.

Comment (DiabolusGX) [reply public/manifest.json]: I tried that color but I think it was too bright and the link embed color will not look good, so I picked lighter one.

Comment (DiabolusGX) [reply public/manifest.json]: Updated the color though

**Review by DiabolusGX — COMMENTED** [review]
- applied suggested changes

**Pull request merged**